srcpkgs/python3-etebase/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'python3-etebase'
+pkgname=python3-etebase
+version=0.31.5
+revision=1
+archs="x86_64"
+wrksrc="etebase-py-${version}"
+build_style=python3-module
+build_helper="rust"
+hostmakedepends="python3-setuptools python3-wheel python3-setuptools-rust cargo"
+makedepends="libsodium libsodium-devel openssl-devel pkg-config"
```

Comment:
libsodium is implied by -devel and pkg-config doesn't make any sense in makedepends